### ISSUE TITLE
Better handling of STATA missings.

### ISSUE THREAD

**Issue by j-bee-pee** [CLOSER-Cohorts/DDI-Lifecycle-Writer-Project#3]
We primarilly use STATA.  Sledgehammer doesn't handle missings well. I have to perform a workaround and stattransfer the .dta files to .spss in order for sledgehammer to properly handle the missing values.

**Comment by spuddybike** [comment]
@j-bee-pee Could you give me an example file? If possible with expected outputs. Also which version of Stata?

**Comment by spuddybike** [comment]
@j-bee-pee 
I don't think we can support every version of all stats packages - so I think I am minded to say - we support up to version x, and if for some reason Stata to SPSS change formats so it is not readable - that you would need to save to a previous version - does this make sense, and are there any issues you might have with the approach?

**Comment by spuddybike** [comment]
@j-bee-pee Could you be more specific, what behaviour do you want? Adrian Dusa has a specific approach we could adopt?  see https://cran.r-project.org/web/packages/DDIwR/DDIwR.pdf recodevalues

**Comment by j-bee-pee** [comment]
@spuddybike Basically Sledgehammer didn't appear to handle missings correctly for STATA files.  I will dig around for an example of what I mean but basically, I have to convert my files to SPSS and loop through each in turn issuing these commands

SPSSINC SELECT VARIABLES MACRONAME="!NumVars" /PROPERTIES TYPE=NUMERIC.
MISSING VALUES !NumVars (-9 thru -1).

so missings (-ve values) are then included as missing counts in the csv file stats correctly.  

.  Also, we send all our files to the UKDS as STATA13 so 'fixing' at a specific stata version would be perfect.

**Comment by spuddybike** [comment]
You need to help me with not handling missing correctly- what do you want the out to be a,b,c ? As per Adrian’s model?
